apps/wallet-mobile/src/AppNavigator.tsx

```diff
@@ -136,7 +137,9 @@ export const AppNavigator = () => {
                 <Stack.Screen name="app-root">
                   {() => (
                     <SearchProvider>
-                      <WalletNavigator />
+                      <TransferProvider>
```

Comment:
lifted up the Transfer provider so we can populate the transfer at anytime (it is always reseted before navigating to in the normal flows) -> QR and Actions Banner.